src/plugins/intel_cpu/tests/functional/single_layer_tests/softmax.cpp

```diff
@@ -190,16 +190,11 @@ const std::vector<SoftMaxConfig> unsupportedConfigsFP32{
      4},
 };
 
-const auto avx512 = CPUSpecificParams{{}, {}, {"jit_avx512"}, "jit_avx512"};
-const auto avx2 = CPUSpecificParams{{}, {}, {"jit_avx2"}, "jit_avx2"};
-const auto sse42 = CPUSpecificParams{{}, {}, {"jit_sse42"}, "jit_sse42"};
-const auto ref = CPUSpecificParams{{}, {}, {"ref_any"}, "ref_any"};
 
-const std::vector<CPUSpecificParams> vecCpuConfigs = {ref, sse42, avx2, avx512};
 const auto OptimizedParams = testing::Combine(testing::Values(ElementType::f32, ElementType::bf16),
                                               testing::ValuesIn(optimizedConfigsFP32),
                                               testing::Values(CommonTestUtils::DEVICE_CPU),
-                                              testing::ValuesIn(filterCPUInfoForDevice(vecCpuConfigs)));
```

Comment:
What is the motivation here?

Reply:
3.1 onednn softmax list:
            CPU_INSTANCE_X64(jit_uni_softmax_fwd_t<avx512_core>)
            CPU_INSTANCE_X64(jit_uni_softmax_fwd_t<avx2>)
            CPU_INSTANCE_X64(jit_uni_softmax_fwd_t<sse41>)  
3.2 softmax list:
CPU_INSTANCE_X64(jit_uni_softmax_fwd_t)

unified jit_uni_softmax_fwd_t.

Reply:
These tests were aimed toward testing our custom modifications to the softmax primitive. Are they removed in this oneDNN version?

Reply:
File https://jira.devtools.intel.com/browse/CVS-114609 to track on this.